ui/Screen/Project/Source/MergeRequestCard.svelte

```diff
@@ -50,7 +54,7 @@
 
 <div class="merge-request-card">
   <div class="left">
-    <Icon.Revision style="margin-right: 0.5rem" />
+    <Icon.Revision style={iconColor} />
```

Comment:
To avoid repeating the styling, this could be replaced by something like the following, with `iconColor` adjusted accordingly:
```suggestion
    <Icon.Revision style={`margin-right: 0.5rem; fill: ${iconColor};`/>
```

Reply:
https://github.com/radicle-dev/radicle-upstream/pull/1721/commits/1c427f08e47bb3015ddfe0ee7c15825129b44d3b